src/installer/__main__.py

```diff
@@ -0,0 +1,83 @@
+"""Installer CLI."""
+
+import argparse
+import os.path
+import re
+import sys
+import sysconfig
+from typing import Dict, Optional, Sequence
+
+import installer
+import installer.destinations
+import installer.sources
+import installer.utils
+
+
+def main_parser() -> argparse.ArgumentParser:
+    """Construct the main parser."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("wheel", type=str, help="wheel file to install")
+    parser.add_argument(
+        "--destdir",
+        "-d",
+        metavar="path",
+        type=str,
+        help="destination directory (prefix to prepend to each file)",
+    )
+    parser.add_argument(
+        "--compile-bytecode",
+        action="append",
+        metavar="level",
+        type=int,
```

Comment:
Could you make it accept a comma-separated list? Right now the help text suggests multiple values can be specified like that, and passing `--compile-bytecode 0 --compile-bytecode 1 --compile-bytecode 2` is a bit long.

Reply:
Perhaps just add a short name, `-O` (eg. `-O0 -O1`).

Reply:
Let's leave this as-is for now -- we can always add a shorter form later. :)